engine/src/main/scala/cromwell/webservice/CromwellApiHandler.scala

```diff
@@ -83,6 +84,34 @@ class CromwellApiHandler(requestHandlerActor: ActorRef) extends Actor {
 
   private def error(t: Throwable)(f: Throwable => RequestComplete[_]): Unit = context.parent ! f(t)
 
+  private def generatePaginationParams(page: Int, pageSize: Int): String = {
+    //uri encode?
```

Comment:
hmm yeah not sure if the url encoding / decoding here and below should be robustified